app/controllers/items/index.js

```diff
@@ -1,75 +1,70 @@
 import config from "../../config/environment";
-import Ember from "ember";
-import searchModule from "../search_module";
+import _ from "lodash";
+import SearchMixin from "../../mixins/search_resource";
```

Comment:
Can we use the absolute path instead of a relative path? I am also not sure, but @abulsayyad123 would you like to give a try?

```suggestion
import SearchMixin from "stock/mixins/search_resource";
```

Reply:
It's working. Added absolute path suggestion.